code/__DEFINES/say.dm

```diff
@@ -31,14 +31,14 @@
 #define MODE_WHISPER_CRIT "whispercrit"
 
 #define MODE_DEPARTMENT "department"
-#define MODE_KEY_DEPARTMENT "h"
-#define MODE_TOKEN_DEPARTMENT ":h"
+#define MODE_KEY_DEPARTMENT "d"
+#define MODE_TOKEN_DEPARTMENT ":d"
 
 #define MODE_ADMIN "admin"
 #define MODE_KEY_ADMIN "p"
 
 #define MODE_DEADMIN "deadmin"
-#define MODE_KEY_DEADMIN "d"
+#define MODE_KEY_DEADMIN "J"
```

Comment:
why j also why ccapitalized that's dumb

Reply:
from a quick admin poll I don't think any of them know about puppet either (I'm gonna remove it.)
that said I have opinions about the renames so uh, please hold

Reply:
> from a quick admin poll I don't think any of them know about puppet either (I'm gonna remove it.) that said I have opinions about the renames so uh, please hold

So did you remove it?!?

Reply:
I uh, did not no

Reply:
WAIT NOOO DON'T REMOVE PUPPET IT'S BASED

Reply:
We just need to advertise its existence better (I use it literally every round I admin)

Reply:
it should be a verb instead of a chat prefix it's confusing and not made clear to jannies

Reply:
As long as it exists in some form then i'm chill

Reply:
ya that seemed fine to me, sorry if i forgot to say such.

Reply:
looks good to me